mysite/wsgi.py

```diff
@@ -0,0 +1,10 @@
+"""WSGI config for mysite project."""
+
+
```

Comment:
You don't really need this many spaces before the imports section. Look at the style in other modules. PEP8 encourages us to be consistent wrt style.

Reply:
Ok.

Reply:
https://github.com/PyCQA/pydocstyle/ - I think it is a good tool for doc check.

Reply:
@serhii73 it's already integrated: https://github.com/kpi-web-guild/django-girls-blog-igorkruglyak/blob/e7ee46c/.pre-commit-config.yaml#L39.
But there's things which automatic tooling just incapable of checking. That's why we need a human review.